meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/imx6ullevk/lmp.cfg

```diff
@@ -46,10 +58,7 @@ CONFIG_FIT_SIGNATURE_STRICT=y
 CONFIG_FIT_VERBOSE=y
 # CONFIG_LEGACY_IMAGE_FORMAT is not set
 CONFIG_PARTITION_UUIDS=y
-CONFIG_FIOVB=y
-CONFIG_CMD_FIOVB=y
```

Comment:
doesn't u-boot-fio-mfgtool need fiovb on a closed board?

Reply:
Fiovb is not really useful without emmc (no rpmb).

Reply:
Actually, we need to leave `CONFIG_FIOVB` enabled regardless of closed state.  We are using this to help with secondary boot path.

(I'll fix)

Reply:
I adjusted these patches to leave `CONFIG_FIOVB` in place.